zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/DataSetInfo.scala

```diff
@@ -22,19 +24,34 @@ object DataSetInfo {
 
   val MetaDataDBName: String = "berry.meta"
   val MetaSchema: Schema = Schema("berry.MetaType",
-                                  Seq(StringField("name")),
-                                  Seq.empty,
-                                  Seq("name"),
-                                  "stats.createTime")
-
-  def parse(json: JsValue): DataSetInfo = {
-    json.validate[DataSetInfo] match {
-      case js: JsSuccess[DataSetInfo] => js.get
+    Seq(StringField("name")),
+    Seq.empty,
+    Seq(StringField("name")),
+    TimeField("stats.createTime"))
+
+  def parse(json: JsValue, schemaMap: Map[String, Schema]): DataSetInfo = {
```

Comment:
would you mind to add scala doc for `parse` and `write` function? sorry, I was lazy at the beginning.